druntime/src/rt/aaA.d

```diff
@@ -559,10 +430,10 @@ extern (C) void* _aaGetX(scope AA* paa, const TypeInfo_AssociativeArray ti,
     immutable hash = calcHash(pkey, aa);
 
     // found a value => return it
-    if (auto p = aa.findSlotLookup(hash, pkey, ti.key))
+    if (auto q = aa.findSlotLookup(hash, pkey, ti.key))
```

Comment:
Need to undo this. I changed the name because I was having segfaults and gdb was confused about what p was.